docs/tutorials/pkg-map.md

```diff
@@ -29,6 +29,23 @@ nixpkgs.overlays = [
 ];
 ```
 
+The user can map package(s) in Nixpkgs to a `pkgconfig-depends` name by
+overlaying the `haskell-nix.extraPkgconfigMappings` attribute:
```

Comment:
re: @yvan-sraka 

> Should we document that directly on the https://input-output-hk.github.io/haskell.nix/ manual?

I added this to the "Nixpkgs overlay" section of this tutorial since it also uses overlays. I guess it's more specifically a `haskell-nix` overlay, but still :thinking: